Sources/Paywalls/Components/PaywallPackagesComponent.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  File.swift
+//  
+//
+//  Created by James Borthwick on 9/5/24.
+//
+
+import Foundation
+// swiftlint:disable missing_docs
```

Comment:
We aren't documenting any APIs at this point (because it could change), that will come as a later step and they'll be easy to find because they're all in #PAYWALLS_COMPONENTS blocks. So in that sense yes they're being tracked.